test/helpers/lightning.ts

```diff
@@ -211,11 +215,22 @@ export const resetLnds = async () => {
 }
 
 export const closeAllChannels = async ({ lnd }) => {
+  let channels
+  try {
+    ;({ channels } = await getChannels({ lnd }))
```

Comment:
what does the `;` do?

Reply:
it's part of the auto formatting we're doing - because we choose some settings a year ago for lint.

I think it's needed whenever we do a deconstruct that have an await. 

not sure why exactly, this is the common pattern in our codebase.